acquia/mc-cs-plugin-custom-objects
# Maut 4196 custom object

If the object is used in a segment filter, alert the user to the segment name and prevent the user from deleting the object
User is alerted via a timed flash message displaying the Segment(s) name

**Step to reproduce**
1. Create custom object
2. Create Segment and use created custom object in to the segment filter
3. Delete custom object
4. Custom object should be deleted
 
**Step to Test**
1. Create custom object
2. Create Segment and use created custom object in to the segment filter
3. Delete custom object
4. Custom object should not be deleted and flash message should be displayed.
5. Flash message should be contain segment name in which custom object is being used.
